src/frontend/src/styles/main.css

```diff
@@ -1156,6 +1160,80 @@ a:hover,
   list-style-type: decimal;
 }
 
+.c-list--recovery {
+  flex-wrap: wrap;
+  counter-reset: recovery;
+  padding: 0;
+  list-style: none;
+}
+
+.c-list--recovery li {
+  position: relative;
+  display: inline-flex;
+  background: var(--rc-input-readonly);
+  color: var(--rc-input-onReadonly);
+  border: var(--vs-line) var(--rc-line) solid;
+
+  border-radius: 999px;
+  padding: 0.5em 0.75em;
+  padding-right: 0.75em;
+  padding-left: 2.9em;
+  width: calc(33.33% - 1rem);
+  margin: 0.5rem;
+  font-size: 0.7em;
+  font-weight: bold;
+  font-family: monospace;
+  text-align: center;
+
+  z-index: 1;
+
+  --current-color: hsl(calc(150 + var(--i) * -180), 100%, 93%);
+}
+
+@media screen and (max-width: 512px) {
+  .c-list--recovery li {
+    width: calc(50% - 1rem);
```

Comment:
out of curiosity, where does `-1rem` come from?

Reply:
its from the padding of the box. (it would be nice to create design tokens for it, but only i we reuse it)